packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -244,10 +243,8 @@ def get_configuration_parameters_with_type_and_default_values(
         configuration_parameters.append(f"{parameter_name}: {parameter_type} = {default_value}")
 
     # Use line separator and spaces to align the parameters appropriately in the generated file.
-    configuration_parameters_with_type_and_value = f",{os.linesep}        ".join(
-        configuration_parameters
-    )
-    parameter_names_as_str = ", ".join(parameter_names)
+    configuration_parameters_with_type_and_value = f",".join(configuration_parameters)
+    parameter_names_as_str = ",".join(parameter_names)
```

Comment:
I don't think generating worse output prior to reformatting with Black is an improvement. Revert this.

Reply:
Updated the separator to ", ".